### DISCUSSION TITLE
Bun 1.0.19 not compiling - Postgres dialect

### DISCUSSION THREAD

**Discussion by michlan-reco** [uptrace/bun#351] [Q&A] [answered]
The origin story to my issue is that I wanted to add usage of bun fixtures to my project, so I ran `go mod vendor` for adding my new dependency and as a side effect, bun version got upgraded from 1.0.15 to 1.0.19.
I am using bun with postgres dialect, when trying to build my project with bun 1.0.19, I get:
`# github.com/uptrace/bun/dialect/pgdialect
vendor/github.com/uptrace/bun/dialect/pgdialect/sqltype.go:56:3: cannot use v (type []string) as type string in return argument`

The compiler is completely right,  seems like in the implementation of `func fieldSQLType(field *schema.Field) string` `field.Tag.Options` used to be a map with value of type string but now it is of type string[] and then we get a return type mismatch.

Has anyone encountered that issue?  I will rollback to 1.0.15 for now but what am I missing?

**Comment by vmihailenco** [comment, marked as answer]
Make sure all bun packages have the same version in go.mod:

```
	github.com/uptrace/bun v1.0.19
	github.com/uptrace/bun/dialect/pgdialect v1.0.19
	github.com/uptrace/bun/driver/pgdriver v1.0.19
	github.com/uptrace/bun/extra/bundebug v1.0.19
	github.com/uptrace/bun/extra/bunotel v1.0.19
```

**Reply by michlan-reco** [reply]
Thanks,  I missed that :)
